src/main/resources/application.yml

```diff
@@ -1,3 +1,13 @@
 micronaut:
   application:
     name: ontology
+    version: 0.1
+storeOntology:
+  bucketSubdirectory: ontology
+  configurationFileName: /configuration
+elasticSearch:
+  servers:
+    - elasticsearch5a1.dsde-dev.broadinstitute.org
+    - elasticsearch5a2.dsde-dev.broadinstitute.org
+    - elasticsearch5a3.dsde-dev.broadinstitute.org
+  indexName: ontology
```

Comment:
This should actually be a generated value, not a config value. When we generate a new ontology index for a particular run, we should probably create a timestamped/versioned index so we don't trample any existing indexes. `ontology` is an existing index on all environments and we don't want to overwrite it accidentally.